app/controllers/admin/imported_commitments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Admin::ImportedCommitmentsController < ActionController::Base
+  layout 'application'
+
+  before_action :user_is_admin
+  before_action :set_commitment, only: [:update]
+
+  def index
+    respond_to do |format|
+      format.json { 
+        @commitments = Commitment.where(commitment_source: 'cbd', cfn_approved: cfn_approved_param) 
+        render json: @commitments.as_json(only: [:id, :name, :description])
+      }
+      format.html {}
```

Comment:
Is this needed?

Reply:
the format/html just renders the view without data, which initialises the vue component so we do need it i think